### DISCUSSION TITLE
general GUI suggestions

### DISCUSSION THREAD

**Discussion by mark-epstein** [bitfocus/companion#2568] [Ideas]
I don't know if these should be feature requests or even guidance for future updates. Would love your thoughts.

1.
![image](https://github.com/bitfocus/companion/assets/80222240/87600efd-bc20-4c58-8485-d378213a5ab6)
My ATEM only supports one M/E and one key, so those drop down options are nonsensical. It would be better if they were greyed out and disabled, or not drop-downs at all.

2. Depending on where you look, a custom variable is referred to as 
`A custom variable (internal:custom_projPower)`
`A custom variable (projPower)`
`$(internal:custom_projPower)`
It would be nice to have a bit more consistency

3.
![image](https://github.com/bitfocus/companion/assets/80222240/26df811b-ff8e-4342-a144-13657c6d2079)
What's the difference between "invert" and switching between `=` and `!=` (or `<` and `>`)? If none, why have both options?

4. It would be nice to have free-form text fields, perhaps one in each button and trigger at least, so I could write notes to myself instead of coming back a year later and wondering, "what was I thinking!!!"

**Comment by Julusian** [comment]
1) I thought it did this, but I guess not.. That should be raised at the atem module repository https://github.com/bitfocus/companion-module-bmd-atem/issues

2) Where are these references? 

3) This is a known side effect of #2549, where we added the invert field to every boolean feedback. Feedbacks are able to opt out of it, which is what should be done here.

4) https://github.com/bitfocus/companion/issues/760

**Reply by mark-epstein** [reply]
1. will do; but I didn't mean to report this, in this discussion area, as a bug. What I was thinking is that any developer of any module could be steered in the direction of not implying options if none exist
2. triggers...conditions, triggers...set variable, buttons/variables. I don't remember which one is which. I'll get and report more details if you like
3. ok
4. I'll admit I just got on a roll suggesting things and didn't research whether this (or the previous one for that matter) was already suggested.
